src/components/LiteracyComponents/ActivitySettingModal.tsx

```diff
@@ -100,7 +100,7 @@ function ActivitySettingModal(props: Props): React.ReactElement {
             </Grid>
             <Grid item style={{width: '100%', paddingTop: '0.5em'}}>
               <Typography style={{fontFamily: 'Arimo'}}>
-                Select {checklistType === 'ReadingTeacher' ? 'or create' : null} a label describing the type of activity that was happening for most of {checklistType === 'ReadingTeacher' ? 'text read during this observation' : 'this observation'}:
+                Select {checklistType === 'ReadingTeacher' ? 'or create' : ''} a label describing the type of activity that was happening for most of {checklistType === 'ReadingTeacher' ? 'text read during this observation' : 'this observation'}:
```

Comment:
Let's change the whitespace application though, my bad :sweat_smile: With this change there still might be a double whitespace appearing - here's a proper solution
```suggestion
                Select {checklistType === 'ReadingTeacher' ? 'or create ' : ' '}a label describing the type of activity that was happening for most of {checklistType === 'ReadingTeacher' ? 'text read during this observation' : 'this observation'}:
```